styleguide/source/_patterns/02-molecules/partner-promo.twig

```diff
@@ -0,0 +1,12 @@
+{% set image = partnerPromo.image %}
+<div class="ama__partner-promo">
+  <div class="ama__partner-promo__image">
+    {% include '@atoms/image/image.twig' %}
```

Comment:
Since this is optional should there be an if statement around the wrapper?

Reply:
Fixed